tests/pass-dep/tempfile.rs

```diff
@@ -0,0 +1,20 @@
+//@ignore-target-windows: no libc on Windows
```

Comment:
This comment seems wrong, the test doesn't use the libc.
```suggestion
//@ignore-target-windows: File handling is not implemented yet
```